insights/combiners/satellite_version.py

```diff
@@ -54,7 +55,7 @@ def _parse_sat_version(version):
     return [major, minor]
 
 
-@combiner(InstalledRpms, optional=[Sat6Ver])
+@combiner([InstalledRpms, SatelliteVersionFromClient], optional=[Sat6Ver])
```

Comment:
I believe you should not put the `SatelliteVersionFromClient` here, since the original `SatelliteVersion` combiner is used to indicate the host is a `"Satellite Server"`. After adding this as optional param, the client of Satellite Server will also be treated as a Sat Server, It will break mand existing rules

Reply:
@xiangce, yes, right, I forgot some rules used it for satellite server only. I'll revert the related code.